readme.md

```diff
@@ -185,6 +185,13 @@ You can use `Reporter` to implement the following features:
 )
   
 ```
+
+#### Enable holmes as pyroscope client
+
+Holmes supports to upload your profile to [pyroscope](https://github.com/pyroscope-io/pyroscope) server. More details
```

Comment:
we need some simple doc, for how to enable this feature.
Assuming that people already have a pyroscope service.